src/applications/static-pages/download-1095b/components/App/index.unit.spec.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import { Provider } from 'react-redux';
-import { render } from '@testing-library/react';
+import { render, waitFor } from '@testing-library/react';
 import { expect } from 'chai';
```

Comment:
Now that there's a loading state before the app is ready, in our unit tests we need to do some waiting for elements to stop existing or to exist before we make an assertion